config/routes.rb

```diff
@@ -288,7 +288,7 @@
   # This is a legacy wild controller route that's not recommended for RESTful applications.
   # Note: This route will make all actions in every controller accessible via GET requests.
 
-  # match ':controller(/:action(/:id))(.:format)'
+  match ':controller(/:action(/:id))(.:format)'
```

Comment:
@david-days , I uncomment this line to avoid test errors. without this route pattern most of the 'get' routes are generating `route miss-match errors`. Please pay your attention for this change.